### ISSUE TITLE
`--fail-under X` where X > 0 should pass on files with no statements

### ISSUE THREAD

**Issue by CobaltCause** [coveragepy/coveragepy#1470]
**Describe the bug**
If I have a file with no statements, `coverage report` correctly says that the coverage is 100%. However, `coverage report --fail-under 100` will fail because it thinks the coverage is 0%.

**To Reproduce**
1. Use Python 3.10
2. Use Coverage 6.4.4 with C extension

```
$ touch test.py
$ coverage run test.py
$ coverage report
Name      Stmts   Miss  Cover
-----------------------------
test.py       0      0   100%
-----------------------------
TOTAL         0      0   100%
$ echo $?
0
$ coverage report --fail-under 100
Name      Stmts   Miss  Cover
-----------------------------
test.py       0      0   100%
-----------------------------
TOTAL         0      0   100%
Coverage failure: total of 0 is less than fail-under=100
$ echo $?
2
```

**Expected behavior**
I would expect this to succeed with no error, it says right above that the coverage is 100%, but then claims it's 0% when `--fail-under` is supplied.

**Comment by nedbat** [comment]
Hmm, I see your point!  For some reason, seven years ago in https://github.com/nedbat/coveragepy/commit/d20a77c9cf1bdba8cf003f24681503313253389b, we added a test specifically to check that an empty file would fail-under=99.   Maybe the behavior of empty files has changed since then? Was it 0% back then?

**Comment by CobaltCause** [comment]
A mystery. One other thing to point out, since it's only in the title, is that this happens with any value for `fail-under` greater than zero. For example, `coverage report --fail-under 1` also causes coverage to exit with code 2.

**Comment by nedbat** [comment]
This is now fixed in commit d5c3daa5.

**Comment by nedbat** [comment]
This is now released as part of [coverage 7.0.0b1](https://pypi.org/project/coverage/7.0.0b1).
